object LogicalError
  1 || 2
end LogicalError
